Zend/zend_builtin_functions.c

```diff
@@ -1656,6 +1660,16 @@ ZEND_FUNCTION(set_error_handler)
 }
 /* }}} */
 
+/* {{{ proto string get_error_handler(void)
+   Returns the currently defined error handler, or null */
+ZEND_FUNCTION(get_error_handler)
+{
```

Comment:
Landing in 3, 2, 1, ... http://git.php.net/?p=php-src.git;a=commitdiff;h=607b7d662a854443713e8a9c651c888527718618